mido/midifiles/midifiles.py

```diff
@@ -315,6 +315,9 @@ def __init__(self, filename=None, file=None,
         elif self.filename is not None:
             with open(filename, 'rb') as file:
                 self._load(file)
+        
```

Comment:
Please add a small comment to explain the rationale behind merging at load time.